test/TestVersioned.py

```diff
@@ -96,6 +96,34 @@ def test_ForMostRecent(self, requests):
         query = requests.getQs()
         self.assertEqual(query["fillerDV"],"10")
 
+    @helpers.TrackRequests
+    def test_ForMUVDateTime(self, requests):
+        url = qs.createVersioned() \
+            .forMarketData([100000001]) \
+            .inAbsoluteDateRange("2021-09-22","2021-09-23") \
+            .inTimeZone("CET") \
+            .inGranularity(Granularity.Day) \
+            .forMUV() \
+            .withFillCustomValue(10) \
+            .execute()
+
+        query = requests.getQs()
+        self.assertEqual(query["fillerDV"],"10")
+
+    @helpers.TrackRequests
+    def test_ForMUVVerionLimit(self, requests):
```

Comment:
this isn't asseting that the expected parameters are present.

Reply:
I'm sure Python has a better way to assert that the generated path is as expected without going char by char. Please make it readable.